### PR TITLE
add public-viewer configuration

### PR DESCRIPTION
**Pull request by filariow**
This PR adds configuration for Public-Viewer as described in [JIRA [ASC-532](https://issues.redhat.com//browse/ASC-532)](https://issues.redhat.com//browse/ASC-532).

### CODE REVIEW

File: pkg/configuration/publicviewer_config.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import (
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+)
+
+type PublicViewerConfig struct {
+	Config *toolchainv1alpha1.PublicViewerConfiguration
+}
+
+func (c PublicViewerConfig) Enabled() bool {
+	return c.Config != nil && c.Config.Enabled
```
Comment (mfrancisc) [inline pkg/configuration/publicviewer_config.go:12]: alternatively you could use:
```suggestion
	return GetBool(c.Config.Enabled, false)
```

File: pkg/test/config/toolchainconfig.go [hunk 2]
```diff
@@ -49,6 +49,22 @@ func PerMemberCluster(name string, value int) PerMemberClusterOptionInt {
 	}
 }
 
+//---PublicViewer Configuration---//
```
Comment (mfrancisc) [inline pkg/test/config/toolchainconfig.go:52]: very minor thing - now it's part of the Host configuration right ?

**Review by mfrancisc — APPROVED** [review]
Looks good! 

Just few cosmetics.

File: pkg/test/config/toolchainconfig.go (hunk 2 above)
Comment (filariow) [reply pkg/test/config/toolchainconfig.go:52]: great catch! let me fix this

**Comment by filariow** [comment]
No more required after moving this configuration under Host (#430)

**Pull request closed without merging**